src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -441,6 +441,7 @@ function New-AzConnectedKubernetes {
         if ($DisableAutoUpgrade) {
             $options += " --set systemDefaultValues.azureArcAgents.autoUpdate=false"
```

Comment:
Does this option get set via ConfigDP now?  Should we be removing this?

Reply:
Will remove in the next commit.